zsl/utils/injection_helper.py

```diff
@@ -77,4 +84,12 @@ def method_or_class_wrapper(*a, **kwargs):
 
 
 def bind(interface, to=None, scope=None):
+    logging.getLogger(__name__).warning("Using this style of binding is deprecated. Use Modules instead.")
     get_current_app().injector.binder.bind(interface, to, scope)
+
+
+def simple_bind(binder, cls, scope):
+    # type: (Binder, Type, Scope) -> None
+    instance = cls()
+    binder.bind(interface=cls, to=instance, scope=scope)
```

Comment:
Python Injector uses a concept of providers, see [docs](https://injector.readthedocs.io/en/latest/terminology.html#provider), I think `ClassProvider` can be used here.